features/dd-sdk-android-trace-otel/build.gradle.kts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+import com.datadog.gradle.config.androidLibraryConfig
+import com.datadog.gradle.config.dependencyUpdateConfig
+import com.datadog.gradle.config.javadocConfig
+import com.datadog.gradle.config.junitConfig
+import com.datadog.gradle.config.kotlinConfig
+import com.datadog.gradle.config.publishingConfig
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+plugins {
+    // Build
+    id("com.android.library")
+    kotlin("android")
+    id("com.google.devtools.ksp")
+
+    // Publish
+    `maven-publish`
+    signing
+    id("org.jetbrains.dokka")
+
+    // Analysis tools
+    id("com.github.ben-manes.versions")
+
+    // Tests
+    id("de.mobilej.unmock")
+    id("org.jetbrains.kotlinx.kover")
+
+    // Internal Generation
+    id("thirdPartyLicences")
+    id("apiSurface")
+    id("transitiveDependencies")
+    id("binary-compatibility-validator")
+}
+
+android {
+    defaultConfig {
+        buildFeatures {
+            buildConfig = true
+        }
+        buildConfigField(
+            "String",
+            "OPENTELEMETRY_API_VERSION_NAME",
+            "\"${libs.versions.openTelemetry.get()}\""
+        )
+        consumerProguardFiles("consumer-rules.pro")
+    }
+    namespace = "com.datadog.android.opentelemetry"
```

Comment:
both namespace and root package should be `com.datadog.android.trace.opentelemetry`, because the common definitions are in `com.datadog.android.trace` . Thus, since it is a subset with a specific implementation belonging to the Trace feature, it should be `com.datadog.android.trace.opentelemetry` imo. This keeps the consistency.